pkg/volume/util/nestedpendingoperations/nestedpendingoperations.go

```diff
@@ -65,6 +65,9 @@ type NestedPendingOperations interface {
 	// IsOperationPending returns true if an operation for the given volumeName and podName is pending,
 	// otherwise it returns false
 	IsOperationPending(volumeName v1.UniqueVolumeName, podName types.UniquePodName) bool
+
+	// GetOperations returns the list of operations
+	GetOperations() []operation
```

Comment:
**If** you are talking about IsOperationPending():

I can't use that because MountVolume() changes the podName to empty before it runs the podName + volumeName as an operation. https://github.com/kubernetes/kubernetes/blob/master/pkg/volume/util/operationexecutor/operation_executor.go#L439

I could however use it to check if an operation is pending for a particular volume but that would defeat the purpose if I pass in an empty string for podName in IsOperationPending().

The idea here is that to check if MountVolume() changes the podName to empty string for attachable volumes and doesn't for non-attachable.